dpnp/dpnp_iface_statistics.py

```diff
@@ -59,7 +60,7 @@
     'min',
     'nanvar',
     'std',
-    'var',
+    'var'
```

Comment:
I would keep comma here. When inserting new string to the end of the list you can accidently forget to add the comma that can be a potential issue.